### PR TITLE
Blog post: Prague 2018 call for volunteers

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

**Comment by thatdocslady** [comment]
Main reviewer for this PR is @mxsasha (for some reason I can't tag you!), but other eyes are welcome to glance...

File: docs/conf/prague/2018/news/call-for-volunteers.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+:template: {{year}}/generic.html
+
+.. post:: June 15, 2018
+   :tags: prague-2018, volunteers, tickets
+
+Ticket Update, Call for Volunteers, Job Fair
+============================================
+
+Greetings, fellow documentarians! The Portland conference is less than three months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
```
Comment (plaindocs) [inline docs/conf/prague/2018/news/call-for-volunteers.rst]: *prague*

File: docs/conf/prague/2018/news/call-for-volunteers.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+**Tickets are selling fast!** We are not sold out yet, but our European community is showing great enthusiasm, and we expect to have another wave of registrations after we announce the presentations soon, so you might want to beat the next rush and `get your ticket now <https://ti.to/writethedocs/write-the-docs-prague-2018>`_.
```
Comment (plaindocs) [inline docs/conf/prague/2018/news/call-for-volunteers.rst]: s/soon/in a week or so/ maybe?

Comment (plaindocs) [reply docs/conf/prague/2018/news/call-for-volunteers.rst]: also remove 'next' from rush?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Looks great

File: docs/conf/prague/2018/news/call-for-volunteers.rst (hunk 1 above)
Comment (thatdocslady) [reply docs/conf/prague/2018/news/call-for-volunteers.rst]: Whoops :)

File: docs/conf/prague/2018/news/call-for-volunteers.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/conf/prague/2018/news/call-for-volunteers.rst]: I'm not sure about "in a week" because that's more specific than "soon"...
Will remove "next".

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
You'll be able to add @mxsasha once she accepts the team invite.

**Comment by mxsasha** [comment]
I just changed my email and added a short paragraph so that we don't discourage people without experience (in a new commit on this branch, so it's part of the PR now). I can imagine the list of tasks can otherwise look a bit daunting. Feel free to tweak my text - the same paragraph is currently in the volunteer application form.

**Review by mxsasha — APPROVED** [review]

**Pull request merged**